autoload/neomake/makers/ft/idris.vim

```diff
@@ -0,0 +1,13 @@
+function! neomake#makers#ft#idris#EnabledMakers()
+    return ['idris']
+endfunction
+
+function! neomake#makers#ft#idris#idris()
+    return {
+        \ 'exe': 'idris',
+        \ 'args': ["--check", "--warn"],
+        \ 'errorformat':
+        \ '%f:%l:%c:,' .
+        \ '%m'
```

Comment:
Are you sure about the errorformat?!
It boils down to `'errorformat': '%f:%l:%c:,%m'` - so two entries whereas the first has no message, and the 2nd one is only a message?!